core/server/worker/src/main/java/alluxio/worker/netty/AbstractReadHandler.java

```diff
@@ -169,7 +169,9 @@ private void validateReadRequest(Protocol.ReadRequest request) throws InvalidArg
   private void setError(Channel channel, Error error) {
     Preconditions.checkNotNull(error);
     try (LockResource lr = new LockResource(mLock)) {
-      if (mContext.getError() != null) {
+      // Note, network errors may be bubbling up through channelUnregistered to reach here before
+      // mContext is initialized.
+      if (mContext == null || mContext.getError() != null) {
```

Comment:
There are other references of `mContext` in the read handler and write handler. Do you think we should add null check for them?